test/swivel/Swivel.sol

```diff
@@ -23,6 +23,8 @@ contract Swivel {
   string constant public NAME = 'Swivel Finance';
   string constant public VERSION = '3.0.0';
   uint256 constant public HOLD = 3 days;
+  /// @dev point in time at which a fee change may take place
+  uint256 public feeChange;
```

Comment:
I like the `@dev` doc here, might be a good idea to expand to other non-obvious contract attributes as well, but that's a future thing.